project/app/src/main/java/com/nexte/nexte/CommentsScene/CommentsView.kt

```diff
@@ -234,6 +261,15 @@ class CommentsView: AppCompatActivity(), CommentsDisplayLogic {
                 itemView.commentBox.text = commentsFormatted.comment
                 itemView.commentDate.text = commentsFormatted.commentDate
                 itemView.playerName.text = commentsFormatted.username
+                itemView.deleteButton.visibility = View.INVISIBLE
```

Comment:
Você poderia colocar essa linha dentro de um else (depois do if da linha 267), pois se o if for verdadeiro essa parte do codigo n precisa ser chamada